src/config/dynamic-theme-fixes.config

```diff
@@ -546,6 +546,9 @@ INVERT
 .ms-Grid-col.ms-sm8.ms-lg4
 .ms-Grid-col.ms-sm4.ms-hiddenMdDown.pullout-textwrap
 .ms-Grid-col.ms-sm2 img
+.pullout-link-text
+.pullout-icon
+span
```

Comment:
Adding span to invert is just not good... I was pretty sure it will break some colors in other places and it even changed the color in menu on the top of the page.
Again I propose some more detailed selector:
`.pullout-textwrap span`